asv.conf.json

```diff
@@ -0,0 +1,70 @@
+{
+    "version": 1,
+    "project": "pydicom",
+    "project_url": "https://github.com/pydicom/pydicom.git",
+
+    // The URL or local path of the repo for the project being benchmarked
+    "repo": ".",
+    "repo_subdir": "",
+
+    // Customizable commands for building, installing, and
+    // uninstalling the project. See asv.conf.json documentation.
+    //
+    // "install_command": ["in-dir={env_dir} python -mpip install {wheel_file}"],
+    // "uninstall_command": ["return-code=any python -mpip uninstall -y {project}"],
+    // "build_command": [
+    //     "python setup.py build",
+    //     "PIP_NO_BUILD_ISOLATION=false python -mpip wheel --no-deps --no-index -w {build_cache_dir} {build_dir}"
+    // ],
+
+    // List of branches to benchmark. If not provided, defaults to "master"
+    // (for git) or "default" (for mercurial).
+    "branches": ["nested-seq-speed", "master"],
+
+    "environment_type": "virtualenv",
+
+    // timeout in seconds for installing any dependencies in environment
+    //"install_timeout": 600,
+
+    // the base URL to show a commit for the project.
+    "show_commit_url": "https://github.com/pydicom/pydicom/commit/",
+
+    // The Pythons you'd like to test against.  If not provided, defaults
+    // to the current version of Python used to run `asv`.
+    // "pythons": ["2.7", "3.6"],
+
+    // The matrix of dependencies to test.
+    "matrix": {
+        "req": {
+            "numpy": []
+        },
+    },
+
+
+    // "benchmark_dir": "benchmarks",
+    "env_dir": "asv/env",
+    "results_dir": "asv/results",
+    "html_dir": "asv/html",
+
+    // The number of characters to retain in the commit hashes.
+    // "hash_length": 8,
+
+    // `asv` will cache results of the recent builds in each
+    // environment, making them faster to install next time.  This is
+    // the number of builds to keep, per environment.
+    // "build_cache_size": 2,
+
+    // The commits after which the regression search in `asv publish`
+    // should start looking for regressions.  ...
+    // "regressions_first_commits": {
+    //    "some_benchmark": "352cdf",  // Consider regressions only after this commit
+    //    "another_benchmark": null,   // Skip regression detection altogether
+    // },
+
+    // The thresholds for relative change in results, after which `asv
+    // publish` starts reporting regressions. ...
+    // "regressions_thresholds": {
+    //    "some_benchmark": 0.01,     // Threshold of 1%
+    //    "another_benchmark": 0.5,   // Threshold of 50%
+    // },
+}
```

Comment:
Good to have that! I only just installed asv, somehow never got to that... 
Is there a possibilty to run this in CI?